### PR TITLE
Add containerized monasca-statsd

### PR DESCRIPTION
**Pull request by zreigz**
PTAL @matrixik @witekest @Brandstetter

### CODE REVIEW

File: monasca-statsd/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+can be minimally run like so:
+
+    docker run -it monasca/statsd:latest
+
+Configuration
+-------------
+
+| Variable                    | Default          | Description                         |
+|-----------------------------|------------------|-------------------------------------|
+| `FORWARDER_URL`             | `http://localhost:17123` | The agent forrwarder url address|
```
Comment (matrixik) [inline monasca-statsd/README.md]: `forrwarder` should be `forwarder`

File: monasca-statsd/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+In environments resembling the official [docker-compose][3] or [Kubernetes][4]
+environments, the image does not require additional configuration parameters and
+can be minimally run like so:
+
+    docker run -it monasca/statsd:latest
+
+Configuration
+-------------
+
+| Variable                    | Default          | Description                         |
```
Comment (matrixik) [inline monasca-statsd/README.md]: Could you format this table to be easier to read in code?
Like using https://www.tablesgenerator.com/markdown_table (Goto `File` => `Paste table data...`)

File: monasca-statsd/start.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/ash
+# shellcheck shell=dash
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
```
Comment (matrixik) [inline monasca-statsd/start.sh]: Add this year and Fujitsu Limited license header

File: monasca-statsd/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+ * `REBULID`: a simple method to invalidate the Docker image cache. Set to
+   `--build-arg REBUILD="$(date)"` to force a full image rebuild.
+ * `HTTP_PROXY` and `HTTPS_PROXY` should be set as needed for your environment
+
+If you'd like to build this image against an uncommitted working tree, consider
+using [git-sync][6] to mirror your local tree to a temporary git repository.
+
+[1]: https://wiki.openstack.org/wiki/Monasca
+[2]: https://github.com/openstack/monasca-statsd
+[3]: https://github.com/hpcloud-mon/monasca-docker/
```
Comment (matrixik) [inline monasca-statsd/README.md]: Change hpcloud-mon to monasca
same for next line

**Comment by matrixik** [comment]
Looks like you need to also comment this name from `ci.py` file for now. It's not finding this service in `docker-compose.yml` file.

**Comment by zreigz** [comment]
I removed this line from `ci.py` After the merge, I will add this and uncomment others.

**Comment by matrixik** [comment]
Merged, thanks for adding this.

**Comment by zreigz** [comment]
Thanks

**Pull request merged**